devscripts/changelog_override.json

```
@@ -68,6 +68,25 @@
     {
         "action": "change",
         "when": "b03fa7834579a01cc5fba48c0e73488a16683d48",
-        "short": "[ie/twitter] Revert 92315c03774cfabb3a921884326beb4b981f786b"
+        "short": "[ie/twitter] Revert 92315c03774cfabb3a921884326beb4b981f786b",
+        "authors": ["pukkandan"]
+    },
+    {
+        "action": "change",
+        "when": "fcd6a76adc49d5cd8783985c7ce35384b72e545f",
+        "short": "[test] Add tests for socks proxies (#7908)",
+        "authors": ["coletdjnz"]
+    },
+    {
+        "action": "change",
+        "when": "db7b054a6111ca387220d0eb87bf342f9c130eb8",
+        "short": "[rh] Add request handler preference framework (#7603)",
```

```suggestion
        "short": "[networking] Add request handler preference framework (#7603)",
```

this one can stay as networking, it's technically not part of the request handlers themselves